typescript/ws-protocol/src/FoxgloveServer.ts

```diff
@@ -284,6 +295,16 @@ export default class FoxgloveServer {
   }
 
   private handleClientMessage(client: ClientInfo, message: ClientMessage): void {
+    const requiredCapability = REQUIRED_CAPABILITY_BY_OPERATION[message.op];
```

Comment:
Should we have a similar check for when the server tries to **send** parameter values or time?

Reply:
I will add that